packages/wallet/api/test/test-lib-wallet.js

```diff
@@ -901,7 +901,7 @@ test('lib-wallet offer methods', async t => {
           description: 'getRefund',
           handle: {
             kind: 'unnamed',
-            petname: 'unnamed-7',
+            petname: 'unnamed-6',
```

Comment:
This change made blindly. Reviewers, how should I figure out whether I should worry about this change?